sonar/modules/users/jsonschemas/users/user-v1.0.0.json

```diff
@@ -2,46 +2,55 @@
   "$schema": "http://json-schema.org/draft-04/schema#",
   "id": "https://sonar.ch/schemas/users/user-v1.0.0.json",
   "additionalProperties": false,
-  "title": "SONAR user v1.0.0",
+  "title": "User",
   "type": "object",
   "properties": {
     "$schema": {
       "type": "string",
       "default": "https://sonar.ch/schemas/users/user-v1.0.0.json"
     },
     "pid": {
-      "title": "User identifier",
-      "description": "User persistent identifier.",
-      "type": "string"
+      "title": "Identifier",
+      "type": "string",
+      "minLength": 1
     },
     "user_id": {
-      "title": "User account identifier",
-      "description": "User account identifier.",
-      "type": "integer"
+      "title": "Account identifier",
+      "type": "integer",
+      "minLength": 1
     },
     "full_name": {
       "title": "Name",
       "type": "string",
-      "minLength": 2
+      "minLength": 1
```

Comment:
Is `minLength=1` property required if `full_name` is already a schema required field ?

Reply:
@jma asked me to add minLength for all properties of type string. I would rather leave this everywhere.